src/common/resource.dto.ts

```diff
@@ -11,11 +11,6 @@ export abstract class Resource {
   @DateTimeField()
   readonly createdAt: DateTime;
 
-  @Field({
-    description: 'Whether the requesting user can delete this resource',
-  })
-  readonly canDelete: boolean;
```

Comment:
This needs to be reverted. The UI uses this to determine if the resource can be deleted or not.